carta/structs.py

```diff
@@ -0,0 +1,16 @@
+import posixpath
```

Comment:
We need a top-level docstring here. For now I suggest `"""This module provides a collection of helper objects for grouping related values together."""`